src/Illuminate/Hashing/ArgonHasher.php

```diff
@@ -164,8 +164,8 @@ protected function time($options)
      * @param $options
```

Comment:
That line is also invalid phpdoc, unrelated to this PR, but might as well be fixed up.